graph-db/src/utils/cmr/__test__/fetchPageFromCmr.test.js

```diff
@@ -0,0 +1,107 @@
+const nock = require('nock')
+const AWS = require('aws-sdk')
+let { getEchoToken } = require('../getEchoToken')
+const { fetchPageFromCMR } = require('../fetchPageFromCmr')
+const { expect } = require('@jest/globals')
+
+// This must be set so '../getEchoToken doesn't
+// attempt to use '../../getSecureParam'
+process.env.IS_LOCAL = true
```

Comment:
Check https://github.com/nasa/earthdata-search/blob/master/serverless/src/decodeId/__tests__/handler.test.js for a better way of setting ENV vars in tests.